src/Task/Git/CommitMessage.php

```diff
@@ -135,6 +138,9 @@ public function run(ContextInterface $context): TaskResultInterface
             );
         }
 
+        if ($this->isMergeCommit($commitMessage)) {
+            return TaskResult::createSkipped($this, $context);
+        }
 
         if ($this->enforceTypeScopeConventions()) {
```

Comment:
you could do the `isMergeCommit` check for merge commits here and skip the type-scope convetions when it's a merge commit. That could make that conventions logic more straight forward